ddd/all_words.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'word'
+
+wolke = Word.new('die', 'Wolke')
+sonne = Word.new('die', 'Sonne')
+katze = Word.new('die', 'Katze')
+kind = Word.new('das', 'Kind')
+wasser = Word.new('das', 'Wasser')
+stern = Word.new('der', 'Stern')
+ofen = Word.new('der', 'Ofen')
```

Comment:
The way the code works now, assigning these local variables is superfluous (they're deleted anyway when the file ends).

Reply:
yep, I know, they're totally useless :D ...I'll delete them